package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "mocha",
-    "start": "node src/index.js"
+    "start": "node src/index.js",
+    "lint": "./node_modules/.bin/eslint src/index.js"
```

Comment:
1, you dont need to use the relative path. simple eslint is enough
2, if a single file is the parameter, then only the file will be checked. We need to check the src dir. PLease chnange the parameter